src/main/java/io/confluent/connect/jdbc/source/TableMonitorThread.java

```diff
@@ -150,6 +162,24 @@ private synchronized boolean updateTables() {
       filteredTables.addAll(tables);
     }
 
+    Map<String, List<TableId>> duplicates = filteredTables.stream()
+        .collect(Collectors.groupingBy(TableId::tableName))
+        .entrySet().stream()
+        .filter(entry -> entry.getValue().size() > 1)
+        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+
+    if (!duplicates.isEmpty()) {
+      log.info("The filtered tables are {} ", filteredTables);
+      log.error("Connector uses unqualified table name as the topic name and has detected "
+          + "duplicate unqualified table names. This could lead to mixed data type and "
+          + "subsequently errors during the processing. Hence to prevent processing errors, "
+          + "JDBC Source connector fails fast fr such configurations. To be able to start the "
+          + "connector, update connector's {} config to include exactly one table in each of the "
+          + "below listed tables.\n {} ", configErrorMsg, duplicates);
```

Comment:
The grammar in this error message needs to be corrected.

> "Duplicate table names found. This connector uses unqualified table names for topic names, and is currently configured with duplicate unqualified table names. Continuing would result in messages from these duplicate tables going to the same topic, and this could lead to mixed records, incompatible schemas, and potential problems in downstream consumers. This connector is stopping to avoid such problems. Change the connector's {} configuration to include exactly one table in each of the tables listed below:\n {}"